config/doctrine.php

```diff
@@ -204,5 +204,22 @@
      */
     'notifications'              => [
         'channel' => 'database'
+    ],
+
+    /*
+     |--------------------------------------------------------------------------
+     | Doctrine Entity Factories
+     |--------------------------------------------------------------------------
+     |
+     |  You can add custom folders with your factories, for instance, if you have
+     |  modular system and each module contains its own factories.
+     |  If no folders are specify, the default one will be used ('database/factories' in project root)
+     |
+     */
+    'factories' => [
+        'folders' => [
+            // 'Modules/User/Database/Factory',
```

Comment:
Can you add `database_path('factories')` as default here.
Also probably better to set the example to `base_path('Modules/...')` ?